Tensile/Source/SolutionMapper.h

```diff
@@ -22,8 +22,16 @@
 
 #pragma once
 
+#if defined(max)
```

Comment:
If we include <cstdlib> here, is std::getenv() not portable to Windows?

Reply:
We can retest this again, it was based on Herve's changes.   I think env changes during test execution were not affecting the cloned environment in a reliable way.   If based on our tests it works we can try again just using std::  there are some envp variations on windows mainin which you get some "frozen" environment variables.   And this disclaimer: "getenv operates only on the data structures accessible to the run-time library and not on the environment "segment" created for the process by the operating system"